src/fswAlgorithms/attControl/mrpProportionalDerivative/_UnitTest/test_mrpProportionalDerivative.py

```diff
@@ -0,0 +1,177 @@
+#
+#  ISC License
+#
+#  Copyright (c) 2024, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
```

Comment:
🔶 The author in all these module's files can be adjusted to "Laboratory for Atmospheric and Space Physics, University of Colorado at Boulder"

Reply:
Done